api/envoy/admin/v2alpha/server_info.proto

```diff
@@ -17,6 +17,10 @@ message ServerInfo {
     LIVE = 0;
     // Server is draining listeners in response to external health checks failing.
     DRAINING = 1;
+    // Server has not yet completed cluster manager initialization.
+    PRE_INITIALIZING = 2;
+    // Server is running the cluster manager initialization callbacks.
```

Comment:
Sorry to be totally pedantic here, but I would make this read:

"Server is running the post cluster manager initialization callbacks (e.g., RDS)."